stores/moderationSubmissionsStore.ts

```diff
@@ -2,18 +2,25 @@ import { gql } from 'graphql-request'
 import { defineStore } from 'pinia'
 import { ref, type Ref } from 'vue'
 import { gqlClient } from '../utils/graphql.js'
-import { type Submission } from '~/typedefs/gqlTypes.js'
+import type { Maybe, HealthcareProfessionalSubmission, Submission } from '~/typedefs/gqlTypes.js'
 
 export enum SelectedSubmissionListViewTab {
     ForReview = 'FOR_REVIEW',
     Approved = 'APPROVED',
     Rejected = 'REJECTED'
 }
 
+export enum SelectedSubmissionType {
+    Facilities = 'FACILITIES',
+    HealthcareProfessionals = 'HEALTHCARE_PROFESSIONALS'
+}
+
```

Comment:
Not clear by the name of the enum. But we want to toggle between 3 things; healthcare professionals, facilities and submissions.

- Healthcare professionals are the hp's existing in our database (so not the healthcare professionals submissions) 
- Facilities are the existing facilities (so not the facility submissions) 
- Submissions the existing submissions